gamearea.py

```diff
@@ -237,6 +237,15 @@ def __motion_cb(self, widget, event):
                 self.over_option = new_option
                 self.redraw()
 
+        elif self.level_data["type"] == GameType.CHOOSE:
+            flag_hover = False
+            for cat in self.choose_option_cats:
+                if event.x > cat.x and event.x < cat.x + cat.width and event.y > cat.y and event.y < cat.y + cat.height:
```

Comment:
This could be made more readable as it looks confusing.

Reply:
Done